### ISSUE TITLE
Typo in 2.2 AAA identifier

### ISSUE THREAD

**Issue by mattgarrish** [w3c/a11y-discov-vocab#194]
In the crosswalk, the identifier in the schema.org column is:

> EPUB Accessibility 1.1 - WCAG 2.1 Level AAA

Should be 2.2.
